app/views/volunteer_ops/index.html.erb

```diff
@@ -16,9 +16,9 @@
       <% @volunteer_ops.each do |op| %>
           <tr>
             <td>
-              <strong><%= link_to op.title, op.link %></strong><br/>
+              <strong><%= link_to_vol_op(op, op.title, op.link) %></strong><br/>
```

Comment:
since we're grabbing title and link from op, doesn't it make more sense just to pass op and hard code the title and link on the other side?